src/Perpetuum.AdminTool/LocalServerRunner.cs

```diff
@@ -355,6 +365,26 @@ private int GetPortFromIni(string gameRoot)
             return portData.ListenerPort;
         }
 
+        private bool GetAdminOnlyModeFromIni(string gameRoot)
+        {
+            var perpetuumIni = Path.Combine(gameRoot, PERPETUUMINIFILE);
```

Comment:
The only nit I'll put here is that the ini file should be read once, by one service, so there is no potential for split/conflicting state to occur.
A very edge-y case for something that is just for logging on a tool we don't use, but still something to consider.

Problem becomes exposing things to the admin tool - and maybe thats messier or causes more violations of design principles.

Reply:
(sorry I meant to leave this in the original review, but didn't commit the comment :( )

Reply:
Nah, good catch! It's bad practice and should be pointed out.

20f674e refactors it, so we read the file once and put the values into a dictionary - and then read the values from the dict instead.

Reply:
Ah yes this, but I also mean that the server itself reads the ini file for GlobalConfig <- that's where the server's copy of this data resides.
TBD if it actually bad/nasty to expose this object to something like a client like the Admintool.  I suspect yes, so I'm cool if basically we assume the state of the ini file will not change between Admintool and server boot